samples/subsys/mgmt/updatehub/src/main.c

```diff
@@ -138,8 +138,7 @@ void main(void)
 	}
 
 #elif defined(CONFIG_MODEM_GSM_PPP)
-	const struct device *uart_dev =
-		DEVICE_DT_GET(DT_BUS(DT_INST(0, zephyr_gsm_ppp)));
+	const struct device *uart_dev = DEVICE_DT_GET_ONE(zephyr_gsm_ppp)
```

Comment:
hmm, this doesn't look correct since you are removing DT_BUS()

Reply:
No that didn't look right.. Fixed now!